common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -1104,7 +1104,7 @@ div.problem {
 
     .notification-message {
       display: inline-block;
-      width: flex-grid(6,10);
+      width: flex-grid(8,10);
```

Comment:
Unfortunately this causes strange wrapping of the button for long languages. Is there no way to just have the message area take up whatever isn't being used by the button? I suppose not since the button is not visible initially. I think you will have to revert this change. :(

![image](https://cloud.githubusercontent.com/assets/484484/25591564/1b44d4d6-2e83-11e7-8605-46229a3775f0.png)